### ISSUE TITLE
Cloud Variables need more command options

### ISSUE THREAD

**Issue by TehTurk** [Neos-Metaverse/NeosPublic#2122]
## Is your tweak request related to a problem? Please describe.

Running multiple commands can be tedious, we should be able to do it faster if we are able.
Also I know Deletion is in the works, but can we rename cloudvars too?

## Describe what would you like tweaked

Right now the typical flow is 

1.>Create
2.>Assign Type
3.>Assign Perms

Ideally I'd like to see

1.>Create, Assign Type, Assign Perms 

So something like:

Creating new Variables:

/createUserVar <path > <type > < default value > < value > < perm_type > < access >
/createGroupVar < group > < path > < type > < default value > < value > < perm_type > < access >

(I didn't put permissions, it's because they aren't permissions they are access types(read, write etc), or access permissions, having 2 objects named permissions can be confusing to some users)

Renaming Variables:

/renameUserVar < path> < newpath >
/renameGroupVar < group > < path > < newpath >

**Comment by Frooxius** [comment]
I'm not sure about this, it might be better handled by building UI for them instead. Building a bunch of different things into a single command would make it a bit unwieldy and really hard to parse. Problem is that you don't set just a single permission, so the command would be something like:

/createGroupVar <group> <path> <type> <default value> <read perm> <write perm> <list perm>

I can add that one if you think that's manageable to use.

Renaming won't likely be supported at all though. If you wanted to rename while preserving all the existing values, that's **a lot** of added complexity to the system. You should make sure you have the naming right when you create them before you start using them. Otherwise it would just be delete and make a new one.

**Comment by Frooxius** [comment]
Added following commands in 2021.4.29.1307:

```
-- /createUserVar <path> <type> <default value> <read perms> <write perms> <list perms>
-- /createGroupVar <group> <path> <type> <default value> <read perms> <write perms> <list perms>
```

**Comment by TehTurk** [comment]
Thanks Froox! And I understand having UI to deal with this might be a bit easier. They are all requests that need to register in some capacity. 

Hmm okay! This is fine for now :) I can wait for the deletion commands.
